lib/cart.rb

```diff
@@ -0,0 +1,27 @@
+require_relative "./item"
+
+class Cart 
+
+    attr_reader :items , :items_bought_list
+
+    def initialize
+        puts "Please enter all the items purchased separated by a comma"
+        @items = gets.chomp.delete(' ').split(',')
+        @items_bought_list = []
+        create_item
+    end
+
+ 
+    def create_item
```

Comment:
Method names are very important.  A method should be named based on what it does, and not what you're using it for. For example, this method could have been named something like get_bought_list or get items_list, or even generate_order_items.